faunadb-common/src/main/java/com/faunadb/common/Connection.java

```diff
@@ -39,6 +39,7 @@
   private static final String API_VERSION = "4";
   private static final int DEFAULT_CONNECTION_TIMEOUT_MS = 10000;
   private static final int DEFAULT_REQUEST_TIMEOUT_MS = 60000;
+  private static final String DEFAULT_USER_AGENT = "Fauna JVM Http Client";
```

Comment:
Great :+1: 

Thank you very much for taking care of it, I took your [commit](https://github.com/fauna/faunadb-jvm/commit/e8b91ef7750cde77a8b18bb28cd253780cc847a7) and applied it here.

I did a tiny change which is to add the word `Jvm` in the header to help us differentiate languages as well.

Reply:
Sadly this breaks the build :cry: 

```
[error] Modules were resolved with conflicting cross-version suffixes in ProjectRef(uri("file:/root/project/"), "faunadb-scala"):
[error]    org.scala-lang.modules:scala-java8-compat _2.11, _2.12
[info]   Compilation completed in 8.982s.
```

Reply:
To fix the cross version issue I have my moved `scala-java8-compat` as a regular dependency.
But then I get
```
[info] faunadb.ClientSpec *** ABORTED ***
[info]   java.lang.NoSuchMethodError: 'void scala.Product.$init$(scala.Product)'
[info]   at com.faunadb.BuildInfo$.<init>(BuildInfo.scala:6)
[info]   at com.faunadb.BuildInfo$.<clinit>(BuildInfo.scala)
[info]   at com.faunadb.BuildInfo.version(BuildInfo.scala)
[info]   at com.faunadb.common.Connection.<clinit>(Connection.java:42)
[info]   at faunadb.FaunaClient$.apply(FaunaClient.scala:49)
[info]   at faunadb.ClientSpec.<init>(ClientSpec.scala:45)
[info]   at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[info]   at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[info]   at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[info]   at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
```
which seems to indicate a Scala version issue.

@lregnier I don't want to spend too much time on this so I am likely to revert the `sbt-buildinfo` commit. You can give it a try before if you want :+1:

Reply:
This seems to do the trick: https://github.com/fauna/faunadb-jvm/pull/297/commits/4711c892abdabda1563e5656ae9e727ee489441b

But I think it might mess up things when publishing according to [sbt: Cross-building](https://www.scala-sbt.org/1.x/docs/Cross-Build.html#Cross+building+with+a+Java+project)

Reply:
Thanks for the investigation, I have decided to remove it for the time being to not block the release further.

Reply:
done :heavy_check_mark: